connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebViewContainer.kt

```diff
@@ -196,6 +200,25 @@ internal class StripeConnectWebViewContainerImpl<Listener : StripeEmbeddedCompon
             controller?.onPageStarted()
         }
 
+        override fun onReceivedHttpError(
+            view: WebView,
+            request: WebResourceRequest,
+            errorResponse: WebResourceResponse
+        ) {
+            controller?.onReceivedError(request.url.toString(), errorResponse.statusCode, errorResponse.reasonPhrase)
+        }
+
+        override fun onReceivedError(view: WebView, request: WebResourceRequest, error: WebResourceError) {
+            // for some reason errorCode and description are only available in API 23+,
```

Comment:
I believe you can still get some sort of message pre API 23 if you use this https://developer.android.com/reference/android/webkit/WebViewClient#onReceivedError(android.webkit.WebView,%20int,%20java.lang.String,%20java.lang.String)

Reply:
Oh nice, good catch! So if you use the deprecated method, you still get the errors 🤔 I'll address this in a fast-follow